src/test/com/yetanalytics/squuid_test.cljc

```diff
@@ -12,19 +12,22 @@
               [cljs.test :refer [deftest testing is]]
               [cljs.spec.test.alpha]
               [com.yetanalytics.squuid :as squuid]
-              [com.yetanalytics.squuid.uuid :refer [compare-uuid]])
+              [com.yetanalytics.squuid.uuid :refer [compare-uuid]]
+              [com.yetanalytics.squuid.time :as t])
              (:require-macros
               [cljs.spec.test.alpha :refer [check instrument]])]))
 
 (deftest squuid-gentest
   (testing "squuid gentests"
     (instrument [`squuid/generate-squuid*
                  `squuid/generate-squuid
-                 `squuid/time->uuid])
+                 `squuid/time->uuid
+                 `squuid/uuid->time])
     (is (every? #(-> % :clojure.spec.test.check/ret :pass?)
                 (check [`squuid/generate-squuid*
                         `squuid/generate-squuid
-                        `squuid/time->uuid])))))
+                        `squuid/time->uuid
+                        `squuid/uuid->time])))))
```

Comment:
With the addition of gentests for `time->uuid` and `uuid->time`, I discovered that generators can't be properly constructed for the `::squuid/timestamp` spec. Here's a solution to that issue I came up with:

```clojure
(s/def ::timestamp
  #?(:clj (s/with-gen (partial instance? java.time.Instant)
            #(sgen/fmap (fn [ts] (t/ms->Instant (inst-ms ts)))
                        (s/gen inst?)))
     :cljs (s/with-gen (partial instance? js/Date)
             #(s/gen inst?))))
```
where `sgen` is `clojure.spec.gen.alpha`.